docs/0.3.5-alpha/basic_syntax/arrays.md

```diff
@@ -1,44 +1,45 @@
+# Ranges
+
+Amber gives you the ability to generate an array of numbers `[Num]` of certain range. There are two types of ranges:
+- `a..b` is exclusive from `a` to `b` excluding `b`.
+- `a..=b` is inclusive from `a` to `b` including `b`.
+
+```ab
+echo 0..10
+// Outputs: 0 1 2 3 4 5 6 7 8 9
+
+echo 0..=10
+// Outputs: 0 1 2 3 4 5 6 7 8 9 10
+```
+
+# Arrays
+
 We learned about array literals in the first chapter. In this chapter we will learn how to make use of them. Arrays are indexed from zero.
 
 To store or retrieve a value at a particular index of an array, you can use the following syntax:
 
 ```ab
-let groceries = ["Apple", "Banana", "Orange"]
-groceries[0] = "Almond"
+let groceries = ["apple", "banana", "cherry", "date"]
+groceries[0] = "kiwi"
 echo groceries[1]
-// Outputs: Banana
+// Outputs: banana
 ```
 
-> WARNING: As of right now (Amber alpha) the subscript syntax does not work with expressions! This means that you can't evaluate expressions like: `foo()[0]` yet.
+> WARNING: As of right now (Amber alpha) the subscript syntax does not work with expressions. This means that you can't evaluate expressions like `foo()[0]` yet.
 
-You can also _echo_ an entire array
+You can also _echo_ an entire array:
 
 ```ab
 echo groceries
-// Outputs: Almond Banana Orange
+// Outputs: kiwi banana cherry date
 ```
 
-To add element to an array you can use the mentioned in the [expressions chapter](/basic_syntax/expressions) addition operator to merge two arrays together.
+To add an element to an array, you can use the mentioned in the [expressions chapter](/basic_syntax/expressions) addition operator to merge two arrays together.
```

Comment:
Is `/basic_syntax/expressions` a valid hyperlink, now we've moved 0.3.5 and 0.4.0 documentation to their own subdirectories?

Reply:
It will resolve to the currently selected documentation version. Everything related to versioning is handled by the website internally so it's a correct URL